numba/cuda/tests/cudadrv/test_cuda_ndarray.py

```diff
@@ -343,6 +318,49 @@ def test_devicearray_contiguous_device_strided(self):
             devicearray.errmsg_contiguous_buffer,
             str(e.exception))
 
+    @skip_on_cudasim('DeviceNDArray class not present in simulator')
+    def test_devicearray_relaxed_strides(self):
+        # From the reproducer in Issue #6824.
+
+        with override_config('NPY_RELAXED_STRIDES_CHECKING', 1):
+            # Construct a device array that is contiguous even though
```

Comment:
Perhaps add something about it being considered contiguous as the stride checking is relaxed?

Reply:
I've added a note in the comment below before the asserts, because this sentence is already full of clauses and tricky to comprehend.